airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -120,9 +120,10 @@ def run(self) -> None:
                 logs_to_skip += 1
             time.sleep(self.fetch_interval.total_seconds())
 
-    def _get_log_events(self, skip: int = 0) -> Generator:
+    def _get_log_events(self, start_from_head=True, skip: int = 0) -> Generator:
```

Comment:
linting

Reply:
also, add typing to `start_from_head`? and change the order of the params to make sure you preserve backwards compatibility? 
`def _get_log_events(self, skip: int = 0, start_from_head: bool = True) -> Generator:`

Reply:
Maybe even make `start_from_head` kwarg-only:

```suggestion
    def _get_log_events(self, skip: int = 0, *, start_from_head: bool = True) -> Generator:
```

This makes signature compatibility easier in the future.